src/Movies.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Movie from './Movie.js';
+import ListGroup from 'react-bootstrap/ListGroup';
+
+class Movies extends React.Component {
+  render() {
+    let renderList = this.props.movies.map((movie, index) => (
+      <Movie key={index} title={movie.title} description={movie.description} /> 
+    ))
+      return (
+        <>
```

Comment:
Here is how I have my image rendering to my application : 

````html
 <Image src={`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_KEY}&center=${this.props.cityData.lat},${this.props.cityData.lon}&zoom=14`} alt="city map" title="Return search map" fluid />
````
What I have done is used the Bootstrap element `<Image>` and passed props I have created and added centering and fluid option for rendering.  Please do not hesitate to reach out if you want a hand.

Reply:
*sidenote* : anytime you want to render more than one element you will need `<>` `</>` tags wrapping your components or the linter screams and your page wont compile right. take it one step further and Wrap your whole app and its rendering components in a `<Container>` and you can ensure that your Application will be dynamic.  The "Container" is not self closing and will need a full closing tag. 

` in my excitement I forgot to mention : your codebase looks great! `